### ISSUE TITLE
All caps triggered in TextField on Windows 10.

### ISSUE THREAD

**Issue by Ryansolar14** [flutter/flutter#87234]
## Steps to Reproduce

1. Hot reload or start the flutter app on a Windows 10 x-64 emulator or installed as a msix.
2. Hit the shift key while typing in the TextField.

**Expected results:** 

The letters that you type while holding down the shift key will be capitalized.
 
**Actual results:** 

Any letters after the shift key was held down are capitalized until the app is restarted or the caps lock key is pressed down. 

<details>
  <summary>Logs</summary>

```
flutter run --verbose
[  +96 ms] executing: [C:\src\flutter\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +119 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms] f4abaa0735eba4dfd8f33f73363911d63931fe03
[   +2 ms] executing: [C:\src\flutter\flutter/] git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[ +198 ms] Exit code 0 from: git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[   +1 ms] 2.2.3
[  +13 ms] executing: [C:\src\flutter\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +123 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms] origin/stable
[        ] executing: [C:\src\flutter\flutter/] git ls-remote --get-url origin
[  +93 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[ +232 ms] executing: [C:\src\flutter\flutter/] git rev-parse --abbrev-ref HEAD
[  +84 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +154 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +9 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +132 ms] executing: C:\Users\Ryan B\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[ +101 ms] List of devices attached
[  +11 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[  +12 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[  +16 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[ +109 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[  +44 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +162 ms] Multiple devices found:
[  +12 ms] Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19042.1110]
[   +4 ms] Edge (web)        • edge    • web-javascript • Microsoft Edge 92.0.902.55
[   +6 ms] [1]: Windows (windows)
[   +2 ms] [2]: Edge (edge)
[  +11 ms] Please choose one (To quit, press "q/Q")
[   +4 ms] : 
[+13914 ms] 1
[  +35 ms] Skipping pub get: version match.
[ +102 ms] Found plugin path_provider at C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider-2.0.2\
[  +12 ms] Found plugin path_provider_linux at
C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.0.2\
[  +12 ms] Found plugin path_provider_macos at
C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_macos-2.0.2\
[  +10 ms] Found plugin path_provider_windows at
C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.0.3\
[ +188 ms] Found plugin path_provider at C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider-2.0.2\
[   +6 ms] Found plugin path_provider_linux at
C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.0.2\
[   +4 ms] Found plugin path_provider_macos at
C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_macos-2.0.2\
[   +9 ms] Found plugin path_provider_windows at
C:\src\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.0.3\
[  +52 ms] Generating C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +189 ms] Initializing file store
[  +18 ms] Skipping target: gen_localizations
[  +13 ms] complete
[   +8 ms] Launching lib\main.dart on Windows in debug mode...
[   +9 ms] C:\src\flutter\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev
C:\src\flutter\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\src\flutter\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter
--debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill
C:\Users\RYANB~1\AppData\Local\Temp\flutter_tools.a9267979\flutter_tool.b59846db\app.dill --packages C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false       
--enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill
build\3c113a45063dc6628e68a4111abcacad.cache.dill.track.dill --enable-experiment=alternative-invalidation-strategy
[  +56 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest
-version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[  +79 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8
-latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64     
Microsoft.VisualStudio.Component.VC.CMake.Project
[   +2 ms] [
             {
               "instanceId": "e26a5a29",
               "installDate": "2021-07-19T15:41:57Z",
               "installationName": "VisualStudio/16.10.3+31424.327",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community",
               "installationVersion": "16.10.31424.327",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\Common7\\IDE\\devenv.exe",      
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2019",
               "description": "Powerful IDE, free for students, open-source contributors, and individuals",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual
               Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-v16.10#16.10.3",        
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2021-07-19T15:41:57.8914789Z",
               "catalog": {
                 "buildBranch": "d16.10",
                 "buildVersion": "16.10.31424.327",
                 "id": "VisualStudio/16.10.3+31424.327",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.10.3",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "3",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.10.3+31424.327",
                 "requiredEngineVersion": "2.10.2174.31177"
               },
               "properties": {
                 "campaignId": "1617375659.1616352441",
                 "canceled": "0",
                 "channelManifestId": "VisualStudio.16.Release/16.10.3+31424.327",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[  +11 ms] Building Windows application...
[  +27 ms] <- compile package:todoity/main.dart
[  +13 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\windows -B build\windows -G Visual Studio 16 2019
[ +225 ms] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
[ +265 ms] -- Configuring done
[ +156 ms] -- Generating done
[  +28 ms] -- Build files have been written to: C:/Users/Ryan B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows
[  +30 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug       
--target INSTALL --verbose
[ +438 ms] Microsoft (R) Build Engine version 16.10.2+857e5a733 for .NET Framework
[   +5 ms] Copyright (C) Microsoft Corporation. All rights reserved.
[ +157 ms] Build started 7/28/2021 3:46:44 PM.
[ +804 ms] Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\INSTALL.vcxproj" on node 1 (default
targets).
[  +45 ms] Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).    
[  +70 ms] InitializeBuildStatus:
[  +62 ms]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +141 ms] CustomBuild:
[  +38 ms]   All outputs are up-to-date.
[ +117 ms]   Checking Build System
[ +420 ms] FinalizeBuildStatus:
[   +5 ms]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +4 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +5 ms] Done Building Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\ZERO_CHECK.vcxproj"    
(default targets).
[ +287 ms] Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).     
[   +9 ms] Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) on node 1      
(default targets).
[   +5 ms] Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\flutter_wrapper_app.vcxproj" 
(4) is building "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\flutter_assemble.vcxproj" (5) onnode 1 (default targets).
[   +3 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.  
[  +34 ms] CustomBuild:
[  +19 ms]   "The build of 'C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\CMakeFiles\6b2694be65aef6f086b78a66ac660d47\flutter_assemble.rule'     
depends on 'C:\USERS\RYAN B\DOCUMENTS\FLUTTER_BOOTCAMP\TODOITY\TODOITY\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is    
produced by the build of 'C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\CMakeFiles\8146a33b388c1cb9a96844b16f1547d3\flutter_windows.dll.rule'. 
The items cannot be built in parallel."
[ +137 ms]   Generating C:/Users/Ryan B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/flutter_windows.dll,
C:/Users/Ryan B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/flutter_export.h, C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/flutter_windows.h, C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/flutter_messenger.h, C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/flutter_plugin_registrar.h, C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/flutter_texture_registrar.h, C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc, C:/Users/Ryan  
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc, C:/Users/Ryan        
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc, C:/Users/Ryan      
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc, C:/Users/Ryan        
B/Documents/Flutter_Bootcamp/todoity/todoity/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_     
[+2916 ms]   [ +129 ms] executing: [C:\src\flutter\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +126 ms]   [ +163 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +4 ms]   [        ] f4abaa0735eba4dfd8f33f73363911d63931fe03
[   +2 ms]   [   +1 ms] executing: [C:\src\flutter\flutter/] git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[ +100 ms]   [ +104 ms] Exit code 0 from: git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[   +1 ms]   [        ] 2.2.3
[   +9 ms]   [  +10 ms] executing: [C:\src\flutter\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +106 ms]   [ +106 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms]   [        ] origin/stable
[   +6 ms]   [        ] executing: [C:\src\flutter\flutter/] git ls-remote --get-url origin
[  +89 ms]   [  +96 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms]   [        ] https://github.com/flutter/flutter.git
[ +133 ms]   [ +135 ms] executing: [C:\src\flutter\flutter/] git rev-parse --abbrev-ref HEAD
[ +114 ms]   [ +114 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms]   [        ] stable
[ +132 ms]   [ +133 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +2 ms]   [   +2 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[  +18 ms]   [   +2 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +8 ms]   [   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms]   [  +15 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +3 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +7 ms]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +3 ms]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[  +35 ms]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[  +26 ms]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +169 ms]   [ +258 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[  +26 ms]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[  +11 ms]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[  +28 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +4 ms]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +2 ms]   [   +4 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +2 ms]   [   +2 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +8 ms]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +5 ms]   [   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[  +13 ms]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +9 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +5 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +27 ms]   [ +155 ms] Initializing file store
[  +59 ms]   [  +59 ms] Skipping target: gen_localizations
[  +17 ms]   [  +17 ms] unpack_windows: Starting due to {}
[+1057 ms]   [+1057 ms] kernel_snapshot: Starting due to {}
[  +15 ms]   [   +6 ms] unpack_windows: Complete
[+5381 ms]   [+5388 ms] C:\src\flutter\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev
C:\src\flutter\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\src\flutter\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --target=flutter --no-print-incremental-dependencies-DFLUTTER_WEB_AUTO_DETECT=true -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --packagesC:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\.dart_tool\package_config.json --output-dill C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\.dart_tool\flutter_build\f72bdb1ee82111470450a061ea9916cc\app.dill --depfile
C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\.dart_tool\flutter_build\f72bdb1ee82111470450a061ea9916cc\kernel_snapshot.d
package:todoity/main.dart
[+39773 ms]   [+39763 ms] kernel_snapshot: Complete
[+2788 ms]   [+2799 ms] debug_bundle_windows_assets: Starting due to {}
[ +540 ms]   [ +540 ms] Manifest contained wildcard assets. Inserting missing file into build graph to force rerun. for more
information see #56466.
[  +47 ms]   [  +48 ms] debug_bundle_windows_assets: Complete
[+1977 ms]   [+1977 ms] Persisting file store
[  +37 ms]   [  +36 ms] Done persisting file store
[  +33 ms]   [  +32 ms] build succeeded.
[  +32 ms]   [  +32 ms] "flutter assemble" took 52,232ms.
[ +267 ms]   [ +267 ms] ensureAnalyticsSent: 255ms
[   +3 ms]   [   +3 ms] Running shutdown hooks
[  +10 ms]   [        ] Shutdown hooks complete
[   +4 ms]   [   +5 ms] exiting with code 0
[ +210 ms] FinalizeBuildStatus:
[   +4 ms]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[   +2 ms]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[   +2 ms] Done Building Project "C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[  +25 ms] InitializeBuildStatus:
[   +2 ms]   Creating "flutter_wrapper_app.dir\Debug\flutter_.2ED629D2.tlog\unsuccessfulbuild" because "AlwaysCreate" was
specified.
[   +9 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[ +216 ms] ClCompile:
[   +2 ms]   All outputs are up-to-date.
[  +95 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[   +2 ms]   flutter_wrapper_app.vcxproj -> C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\Debug\flutter_wrapper_app.lib
[  +50 ms] FinalizeBuildStatus:
[   +2 ms]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.2ED629D2.tlog\unsuccessfulbuild".
[   +5 ms]   Touching "flutter_wrapper_app.dir\Debug\flutter_.2ED629D2.tlog\flutter_wrapper_app.lastbuildstate".
[   +3 ms] Done Building Project "C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[ +177 ms] Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (6) on node 1   
(default targets).
[   +4 ms] InitializeBuildStatus:
[   +2 ms]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.7D6D3296.tlog\unsuccessfulbuild" because "AlwaysCreate" was      
specified.
[   +9 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +36 ms] ClCompile:
[   +1 ms]   All outputs are up-to-date.
[  +18 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[   +2 ms]   flutter_wrapper_plugin.vcxproj -> C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[  +21 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.7D6D3296.tlog\unsuccessfulbuild".
[   +3 ms]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.7D6D3296.tlog\flutter_wrapper_plugin.lastbuildstate".
[   +4 ms] Done Building Project "C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[ +140 ms] Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\runner\todoity.vcxproj" (7) on node 1 (default targets).[   +3 ms] InitializeBuildStatus:
[   +2 ms]   Creating "todoity.dir\Debug\todoity.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +11 ms] CustomBuild:
[   +4 ms]   All outputs are up-to-date.
[ +106 ms] ClCompile:
[   +2 ms]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\bin\HostX64\x64\CL.exe /c
/I"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\windows" /I"C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\windows\flutter\ephemeral" /I"C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\windows\flutter\ephemeral\cpp_client_wrapper\include" /Zi /nologo /W4 /WX
/diagnostics:column /Od /Ob0 /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE /D _UNICODE /D
"CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR     
/std:c++17 /Fo"todoity.dir\Debug\\" /Fd"todoity.dir\Debug\vc142.pdb" /external:env:EXTERNAL_INCLUDE /external:W4 /Gd /TP /wd4100  
/errorReport:queue "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\windows\flutter\generated_plugin_registrant.cc"    
"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\windows\runner\flutter_window.cpp"
[ +389 ms]   generated_plugin_registrant.cc
[+1223 ms]   flutter_window.cpp
[+2372 ms]   Generating Code...
[ +395 ms] ResourceCompile:
[   +1 ms]   All outputs are up-to-date.
[ +166 ms] Link:
[   +1 ms]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\bin\HostX64\x64\link.exe
/ERRORREPORT:QUEUE /OUT:"C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\runner\Debug\todoity.exe"       
/INCREMENTAL /ILK:"todoity.dir\Debug\todoity.ilk" /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib "C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\windows\flutter\ephemeral\flutter_windows.dll.lib" kernel32.lib user32.lib gdi32.lib 
winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker'      
uiAccess='false'" /manifest:embed /manifestinput:"C:\Users\Ryan
B\Documents\Flutter_Bootcamp\todoity\todoity\windows\runner\runner.exe.manifest" /DEBUG /PDB:"C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/todoity.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE     
/NXCOMPAT /IMPLIB:"C:/Users/Ryan B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/todoity.lib" /MACHINE:X64/machine:x64 todoity.dir\Debug\Runner.res
[   +7 ms]   todoity.dir\Debug\flutter_window.obj
[   +4 ms]   todoity.dir\Debug\main.obj
[   +2 ms]   todoity.dir\Debug\run_loop.obj
[   +1 ms]   todoity.dir\Debug\utils.obj
[   +1 ms]   todoity.dir\Debug\win32_window.obj
[   +2 ms]   todoity.dir\Debug\generated_plugin_registrant.obj
[ +742 ms]   todoity.vcxproj -> C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\runner\Debug\todoity.exe
[  +22 ms] FinalizeBuildStatus:
[   +2 ms]   Deleting file "todoity.dir\Debug\todoity.tlog\unsuccessfulbuild".
[   +5 ms]   Touching "todoity.dir\Debug\todoity.tlog\todoity.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\runner\todoity.vcxproj"(default targets).
[  +25 ms] InitializeBuildStatus:
[   +2 ms]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +20 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[ +107 ms]   Building Custom Rule C:/Users/Ryan B/Documents/Flutter_Bootcamp/todoity/todoity/windows/CMakeLists.txt
[ +142 ms] FinalizeBuildStatus:
[   +2 ms]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[   +3 ms]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\ALL_BUILD.vcxproj"     
(default targets).
[  +28 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +194 ms] PostBuildEvent:
[   +1 ms]   setlocal
[   +1 ms]   "C:\Program Files (x86)\Microsoft Visual
Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake 
[   +3 ms]   if %errorlevel% neq 0 goto :cmEnd
[   +2 ms]   :cmEnd
[   +1 ms]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[   +1 ms]   :cmErrorLevel
[   +1 ms]   exit /b %1
[   +1 ms]   :cmDone
[   +1 ms]   if %errorlevel% neq 0 goto :VCEnd
[   +3 ms]   :VCEnd
[ +126 ms]   -- Install configuration: "Debug"
[  +18 ms]   -- Up-to-date: C:/Users/Ryan B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/icudtl.dat
[   +2 ms]   -- Up-to-date: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/flutter_windows.dll
[  +28 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets
[   +6 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +7 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[   +3 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/fonts
[   +9 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf       
[  +12 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[ +225 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/NOTICES
[  +10 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/packages
[   +6 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[   +4 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets       
[   +4 ms]   -- Installing: C:/Users/Ryan
B/Documents/Flutter_Bootcamp/todoity/todoity/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[  +38 ms] FinalizeBuildStatus:
[   +3 ms]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[   +8 ms]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[   +4 ms] Done Building Project "C:\Users\Ryan B\Documents\Flutter_Bootcamp\todoity\todoity\build\windows\INSTALL.vcxproj"       
(default targets).
[   +2 ms] Build succeeded.
[   +2 ms]     0 Warning(s)
[   +2 ms]     0 Error(s)
[   +4 ms] Time Elapsed 00:01:06.07
[  +62 ms] Building Windows application... (completed in 67.5s)
[+7324 ms] Observatory URL on device: http://127.0.0.1:65400/ShTUZXrwdDA=/
[  +32 ms] Caching compiled dill
[ +276 ms] Connecting to service protocol: http://127.0.0.1:65400/ShTUZXrwdDA=/
[ +428 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at
http://127.0.0.1:65400/ShTUZXrwdDA=/.
[ +168 ms] DDS is listening at http://127.0.0.1:65406/7cAJC0CUK4U=/.
[ +186 ms] Successfully connected to service protocol: http://127.0.0.1:65400/ShTUZXrwdDA=/
[  +59 ms] DevFS: Creating new filesystem on the device (null)
[  +34 ms] DevFS: Created new filesystem on the device (file:///C:/Users/RYANB~1/AppData/Local/Temp/todoity33b687e8/todoity/)
[  +24 ms] Updating assets
[ +328 ms] Manifest contained wildcard assets. Inserting missing file into build graph to force rerun. for more information see
#56466.
[ +395 ms] Syncing files to device Windows...
[  +54 ms] <- reset
[  +51 ms] Compiling dart to kernel with 0 updated files
[  +35 ms] <- recompile package:todoity/main.dart f5b8ce42-449d-4dc3-9caa-81da9082de38
[  +29 ms] <- f5b8ce42-449d-4dc3-9caa-81da9082de38
[ +364 ms] Updating files.
[  +76 ms] DevFS: Sync finished
[  +17 ms] Syncing files to device Windows... (completed in 573ms)
[  +33 ms] Synced 0.0MB.
[  +15 ms] <- accept
[  +73 ms] Connected to _flutterView/0x1c38ab14810.
[ +113 ms] Flutter run key commands.
[ +143 ms] r Hot reload. 
[ +175 ms] R Hot restart.
[  +42 ms] h Repeat this help message.
[  +19 ms] d Detach (terminate "flutter run" but leave application running).
[   +4 ms] c Clear the screen
[  +29 ms] q Quit (terminate the application on the device).
[  +22 ms]  Running with sound null safety 
[  +27 ms] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:65406/7cAJC0CUK4U=/
[+1433 ms] The Flutter DevTools debugger and profiler on Windows is available at:
                    http://127.0.0.1:9104?uri=http%3A%2F%2F127.0.0.1%3A65406%2F7cAJC0CUK4U%3D%2F

```

```
No issues found! (ran in 10.4s)
```

```
flutter doctor -v
[√] Flutter (Channel stable, 2.2.3, on Microsoft Windows [Version 10.0.19042.1110], locale en-US)
    • Flutter version 2.2.3 at C:\src\flutter\flutter
    • Framework revision f4abaa0735 (4 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\Ryan B\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[X] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.10.3)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.10.31424.327
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)

[√] VS Code (version 1.58.2)
    • VS Code at C:\Users\Ryan B\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.24.0

[√] Connected device (2 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19042.1110]
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 92.0.902.55

! Doctor found issues in 2 categories.
```

</details>

**Comment by TahaTesser** [comment]
Hi @Ryansolar14 
This was reported in https://github.com/flutter/flutter/issues/75675 
I can confirm it is fixed, currently works in the latest `dev` and `master` channels

Can you please upgrade to the latest  `dev` and try again if you want
Execute the following commands to upgrade
`flutter channel dev`
`flutter upgrade --force`
`flutter doctor -v`

Closing as duplicate of https://github.com/flutter/flutter/issues/75675
